jsonrpc/specification.md

```diff
@@ -0,0 +1,49 @@
+# Portal JSON-RPC specification
+
+The `*` is meant to be a placeholder to substitute different Portal sub-networks `history`, `state`, `beacon`, etc
```

Comment:
something like `<network-identifier>` or `<identifier>` or `<namespace>` might be more visually clear.